src/site/content/en/blog/gedi/index.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: |
+  GEDI Digital reduced mobile bounce rate by 8% with Core Web Vitals
+subhead: |
+  By focusing development effort on performance improvement and applying the
+  latest web technologies and techniques, GEDI Digital has created one of the
+  fastest newspaper sites.
+authors:
+  - maurizioparadisi
+date: 2021-12-10
+hero: image/VbsHyyQopiec0718rMq2kTE1hke2/qg8rHQ5qlKK9t4Z6qTH2.jpeg
```

Comment:
This hero image has no corresponding alt text. I believe it can be added by adding an `alt` key to the front matter.